### PR TITLE
Update Documentation.

### PR DESCRIPTION
**Pull request by MaedahBatool**
I have improved the readme.md file a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,25 +7,25 @@ We know how important fundraising online is to accomplish your goals. For many n
...
 
 There's a better way. Now you have Give.
 
-### Simple and Pain Free Giving ###
+### Simple and Pain-Free Giving ###
 
-Give works great right in WordPress in a way that you're already familiar in working. Simply install and activate the plugin to get started. The backbone of Give lies within its forms. Forms are what allow you to accept donations anywhere on your website. When creating a form you may specify whether the form is a single or multi-level amount. As well, forms can accept a user provided donation amount.
+Give works great right in WordPress in a way that you're already familiar with working. Simply install and activate the plugin to get started. The backbone of Give lies within its forms. Forms are what allow you to accept donations anywhere on your website. When creating a form, you may specify whether the form is a single or multi-level amount. As well, forms can accept a user provided donation amount.
```
Comment (DevinWalker) [inline README.md]: Looks good

File: README.md [hunk 2]
```diff
@@ -7,25 +7,25 @@ We know how important fundraising online is to accomplish your goals. For many n
...
 
 ### Zero Commission Charges ###
 
 We earn money by selling add-ons, not from the money you raise. What you raise is yours to keep to support your cause. Period. Add-ons are premium features that enable you to extend the functionality of Give. For example, you can use one of our add-ons to accept funds through your favorite payment gateway.
 
 ### Easy to Customize and Enhance ###
 
-Give is built by [WordImpress](https://wordimpress.com "Visit the the developers of Give - WordImpress"), our plugins are built from the ground up to be intuitive and easy to use. Along with the plugin is a support team that is dedicated, fast to respond, and always willing to squash bugs and help troubleshoot.
+Give is built by [WordImpress](https://wordimpress.com "Visit the the developers of Give - WordImpress"), our plugins are built from the ground up to be intuitive and easy to use. Along with the plugin is a support team that is dedicated, fast to respond, and always willing to squash bugs and helps troubleshoot.
```
Comment (DevinWalker) [inline README.md]: This should actually just be `help` not `helps`.

Comment (MaedahBatool) [inline README.md]: Oh yes....my bad.

**Comment by MaedahBatool** [comment]
Hey @DevinWalker,
Thanks approving the changes which I suggested. Yeah sure, I'll be readme.txt file as well. :)

**Comment by DevinWalker** [comment]
@MaedahBatool - don't worry about it. We're in the process of heavily revising it now internally.

**Pull request merged**